### ISSUE TITLE
Czech locale export is missing in locale/index.js

### ISSUE THREAD

**Issue by RakhmatovR** [date-fns/date-fns#1225]
So, when I'm trying to import Czech locale like that:
```
import { cs } from 'date-fns/locale';
```
It's okay, no warnings displayed in editor, because of TypeScript definition, which has this export. But in runtime I got a warning.
```
export 'cs' was not found in 'date-fns/locale'
```
Seems like missing export in `src/locale/index.js`

**Comment by kossnocorp** [comment]
Omg, we just can't ship this locale, this is our 3rd attempt 😱

We'll fix that in the next release that is coming soon. I'll keep you updated.

**Comment by kossnocorp** [comment]
It was fixed in `v2.0.0-beta.1`.
